x/staking/keeper/epoch.go

```diff
@@ -0,0 +1,91 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	proto "github.com/gogo/protobuf/proto"
+)
+
+// keys
+var (
+	NextEpochActionIDPrefix = []byte("next_epoch_action_id")
```

Comment:
It's not really a prefix, right?  It's just the key at which the next ID is stored.

Let's just call it `NextEpochActionID`

Reply:
modified